representative/models.py

```diff
@@ -1423,9 +1433,10 @@ def update_representative_twitter_details(self, representative, twitter_dict,
         }
         return results
 
-    def reset_representative_image_details(self, representative, twitter_profile_image_url_https,
-                                             twitter_profile_background_image_url_https,
-                                             twitter_profile_banner_url_https):
+    @staticmethod
+    def reset_representative_image_details(representative, twitter_profile_image_url_https,
```

Comment:
If all variables can't fit on one line, please break each one onto it's own line.